Source/Core/Core/ConfigManager.cpp

```diff
@@ -62,6 +62,11 @@ static const struct
 	{ "Wiimote4Connect",    347 /* WXK_F8 */,     1 /* wxMOD_ALT */ },
 	{ "BalanceBoardConnect",348 /* WXK_F9 */,     1 /* wxMOD_ALT */ },
 #endif
+
+	{ "VolumeUp",            0,                    0 /* wxMOD_NONE */ },
+	{ "VolumeDown",          0,                    0 /* wxMOD_NONE */ },
+	{ "VolumeToggleMute",    0,                    0 /* wxMOD_NONE */ },
```

Comment:
Vertical alignment of the `g_HKData` struct is a mess at the moment. I will do a separate PR, fixing all entries, once this one is merged.